spec/lib/problem_check/twitter_login_spec.rb

```diff
@@ -41,5 +42,31 @@
         )
       end
     end
+
+    context "when Twitter authentication seems to be permanently broken" do
+      before do
+        authenticator.stubs(:healthy?).returns(false)
+        tracker.stubs(:blips).returns(3)
```

Comment:
No need to stub the blips, you can just set this when you initialize the tracker. A new `let` in this `context` should do the trick